src/exec.js

```diff
@@ -178,6 +188,15 @@ function execAsync(cmd, opts, callback) {
 //@   console.log('Exit code:', code);
 //@   console.log('Program output:', output);
 //@ });
+//@
+//@ var child = exec('cut -d\\; -f1 some_huge_file.csv', {
```

Comment:
Could we use a separator other than `;` here? It would just make the example cleaner to use something that doesn't need to be escaped. Not a big concern though.